src/plugins/kibana/public/dashboard/index.html

```diff
@@ -1,7 +1,62 @@
 <div dashboard-app class="app-container dashboard-container">
-  <navbar ng-show="chrome.getVisible()" name="dashboard">
-    <span class="name" ng-if="dash.id" ng-bind="::dash.title" tooltip="{{::dash.title}}" tooltip-append-to-body="true"></span>
+  <navbar name="dashboard-options" class="kibana-nav-options">
+    <div class="kibana-nav-info">
+      <span ng-show="dash.id" class="kibana-nav-info-title">
+        <span ng-bind="::dash.title"></span>
+      </span>
+    </div>
+
+    <div class="button-group kibana-nav-actions" role="toolbar">
+      <button ng-click="newDashboard()"
+        aria-label="New Dashboard">
+        <span>New</span>
+      </button>
+      <button
+        aria-label="Save Dashboard"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('save') }}"
+        ng-class="{active: configTemplate.is('save')}"
+        ng-click="configTemplate.toggle('save');">
+        <span>Save</span>
+      </button>
+      <button
+        aria-label="Load Saved Dashboard"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('load') }}"
+        ng-class="{active: configTemplate.is('load')}"
+        ng-click="configTemplate.toggle('load');">
+        <span>Open</span>
+      </button>
+      <button
+        aria-label="Share Dashboard"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('share') }}"
+        ng-class="{active: configTemplate.is('share')}"
+        ng-click="configTemplate.toggle('share');">
+        <span>Share</span>
+      </button>
+      <button
+        aria-label="Add Visualization"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('pickVis') }}"
+        ng-class="{active: configTemplate.is('pickVis')}"
+        ng-click="configTemplate.toggle('pickVis');">
+        <span>Add visualization</span>
+      </button>
+      <button
+        aria-label="Options"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('options') }}"
+        ng-class="{active: configTemplate.is('options')}"
+        ng-click="configTemplate.toggle('options');">
+        <span>Options</span>
+      </button>
+      <div class="chrome-actions"kbn-chrome-append-nav-controls></div>
```

Comment:
The spinner needs to be moved out of `kbn-chrome-append-nav-controls` and into the overall chrome otherwise it will be missing in any application that doesn't use `kbn-chrome-append-nav-controls`.